lib/querystring.js

```diff
@@ -115,11 +116,6 @@ function qsUnescape(s, decodeSpaces) {
   }
 }
 
-
-const hexTable = [];
-for (var i = 0; i < 256; ++i)
-  hexTable[i] = '%' + ((i < 16 ? '0' : '') + i.toString(16)).toUpperCase();
-
 // These characters do not need escaping when generating query strings:
```

Comment:
I think placing private bits into`internal` would make the boundaries clearer, whether its shared or not, like `BufferList` for `_stream_readable.js` (only used once there).